news/2020-01-03-monthly-beatmapping-contests-return.md

```diff
@@ -0,0 +1,135 @@
+</audio>
+
+#### Kiradoki Harmony (145 BPM)
+
+Beatmap template download: [Full version](https://assets.ppy.sh/artists/62/osz/MYLK%20-%20Kiradoki%20Harmony.osz) (4:06) / [Short version](https://assets.ppy.sh/contests/84/files/MYLK_-_Kiradoki_Harmony.osz) (2:30)
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/62/previews/1759.mp3" type="audio/mpeg">
+</audio>
+
+#### Prism (100 BPM)
+
+Beatmap template download: [Full version](https://assets.ppy.sh/artists/62/osz/MYLK%20-%20Prism.osz) (4:00) / [Short version](https://assets.ppy.sh/contests/84/files/MYLK_-_Prism.osz) (2:20)
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/62/previews/1754.mp3" type="audio/mpeg">
+</audio>
+
+#### Daydreamer (170 BPM)
+
+Beatmap template download: [Full version](https://assets.ppy.sh/artists/62/osz/MYLK%20-%20Daydreamer.osz) (4:10) / [Short version](https://assets.ppy.sh/contests/84/files/MYLK_-_Daydreamer.osz) (2:40)
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/62/previews/1748.mp3" type="audio/mpeg">
+</audio>
+
+#### Let You Go (140 BPM)
+
+Beatmap template download: [Full version](https://assets.ppy.sh/artists/62/osz/MYLK%20-%20Let%20You%20Go.osz) (3:00)
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/62/previews/1752.mp3" type="audio/mpeg">
+</audio>
+
+All beatmap templates above include shortened versions of songs for the sake of better beatmapping.
+
+## <a id="screening"></a>Screening
+
+While most beatmapping contests are judged solely by a panel of 3-5 judges, that isn't the case for the Monthly Beatmapping Contest.
+
+We've selected **10** experienced beatmappers to provide their input and decide which 5 submissions proceed to community voting.
+
+The week-long process of narrowing down submissions involves each member of the screening crew providing their top 3 entries in order of preference. The highest quality entry receives a value of 3, the next highest a value 2, and the final entry a value of 1. The top 5 are determined by the cumulative value of each entry.
+
+For transparency's sake, the screening users this month are [ScubDomino](https://osu.ppy.sh/users/8972308), [Noffy](https://osu.ppy.sh/users/1541323), [Zelq](https://osu.ppy.sh/users/8953955), [Uberzolik](https://osu.ppy.sh/users/1314547), [Venix](https://osu.ppy.sh/users/5999631), [Smokeman](https://osu.ppy.sh/users/2140676), [Sing](https://osu.ppy.sh/users/3795679), [Cris-](https://osu.ppy.sh/users/6175280), [Realazy](https://osu.ppy.sh/users/918297), and [Nuvolina](https://osu.ppy.sh/users/10974170). In case any of them drop out, back-up screening users are [Myxo](https://osu.ppy.sh/users/2202645), [Cheri](https://osu.ppy.sh/users/5226970), and [Sinnoh](https://osu.ppy.sh/users/4236057). All of these users will be reviewing the submissions without knowledge of who their creators are, and each of their preferred submissions will be viewable when the top 5 entries are announced.
```

Comment:
```suggestion
For transparency's sake, the screening users this month are  [Cris-](https://osu.ppy.sh/users/6175280), [Noffy](https://osu.ppy.sh/users/1541323),  [Nuvolina](https://osu.ppy.sh/users/10974170), [Realazy](https://osu.ppy.sh/users/918297), [ScubDomino](https://osu.ppy.sh/users/8972308), [Sing](https://osu.ppy.sh/users/3795679), [Smokeman](https://osu.ppy.sh/users/2140676), [Uberzolik](https://osu.ppy.sh/users/1314547), [Venix](https://osu.ppy.sh/users/5999631), and [Zelq](https://osu.ppy.sh/users/8953955). 

In case any of them drop out, back-up screening users are [Cheri](https://osu.ppy.sh/users/5226970), [Myxo](https://osu.ppy.sh/users/2202645), and [Sinnoh](https://osu.ppy.sh/users/4236057). 
All of these users will be reviewing the submissions without knowledge of who the creators are, and each of their preferred submissions will be view-able when the top 5 entries are announced.
```

Reply:
Placed into alphabetical order